### DISCUSSION TITLE
Ideas for Presentify App

### DISCUSSION THREAD

**Discussion by rampatra** [Softal-io/presentify-community#1] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place where users of Presentify can request/upvote new features, and see the progress of ongoing development.

**Comment by nicolaikrueger** [comment]
Hey, I would like to combine my Elgato Stream Deck with the Presentify App. Anyone already on it? Cheers.

**Reply by rampatra** [reply]
Yes, you can easily configure the key shortcuts on Stream Deck. I have done it in the past with no issues. 

For example, if you want to configure a button on Stream Deck to start the annotation, you set it as `Ctrl + A`. And, if you want to set a button for a specific Annotation mode like Rectangles then you can set it as `Ctrl + A + R`. `Ctrl + A` is to start Presentify and then `R` for the Rectangle mode.

**Comment by WaffleJim** [comment]
I've used Presentify over the past year, as I teach online and do demos in Photoshop and Toon Boom. I'd love to have a feature that displays keyboard shortcuts I use as I work. Currently my only option for this is to use ScreenFlow. But this is only good for prerecorded demos and still has it's limits.

A perfect solution, in my eyes, is one that displays the key combos but also allows me to display the Windows equivalent keys I use on my Mac. For instance, Macs use Command (⌘) but Windows uses Control. If this could be something I use live, while demoing to my students it would be amazing! Thanks!

**Reply by rampatra** [reply]
Yes, definitely. This is in the pipeline. Thanks for requesting.

**Reply by rampatra** [reply]
Sorry for not updating here but we now have a different app that lets you show key presses on screen. Check out [KeyScreen](https://keyscreenapp.com/). 

<img width="2880" height="1800" alt="1-keypresses" src="https://github.com/user-attachments/assets/70d25b5d-2588-4639-bc19-54b9a644e4c9" />

**Comment by lennybAudioEngineer** [comment]
Love the app. I would love a feature that dissolves the latest annotation after a period of time (like 3 to 5 seconds). Zoom has this feature and calls it "Laser Pointer."  I'm sorry if someone has already requested this. Hope to see it in one of the updates!  Thanks

**Reply by rampatra** [reply]
This feature is already present. Please go to Settings -> Annotate and then check the Auto-erase annotation option. You can set the auto erase time as well.

**Comment by lennybAudioEngineer** [comment]
Thanks!  I'm very happy!

**Comment by dburchill** [comment]
Great app, I use it daily since discovering it. 

I would really like an eraser. In the course of teaching a lesson, I often need to erase just part of a drawing, for example, if I'm teaching about linked lists, I want to erase and redraw lines to show how the pointers get adjusted. Right now, I have to erase everything and redraw my whole diagram. 

I find I can't set pen colors dark enough, I have the opacity set to 100, but they still look more like highlighters than pens, could we have the ability to make the pens more opaque.

**Reply by rampatra** [reply]
You can erase selected drawings instead of everything. Please watch this video to see how - https://youtu.be/-fRLYxp7Ak4?si=JBsa5Ym1geSNVOiH

Regarding the opacity, from what I am reading, it seems you have enabled the Highlight mode unknowingly. Please, press H to disable it (when in annotation mode). This will give you 100% opaque drawings.

**Reply by jorgejkoechlin** [reply]
Have tried all and cursor still shadowy grey not bold enough

**Reply by jorgejkoechlin** [reply]
OK OK got it pressed H out of nowhere and did it

**Comment by dburchill** [comment]
Hi Ram,
Thanks for the prompt reply, it was in highlight mode. I was sure it was something simple like that, but I couldn’t find it.
Cheers,
Dave.

From: Ram ***@***.***>
Date: Sunday, September 21, 2025 at 6:12 AM
To: Softal-io/presentify-community ***@***.***>
Cc: Burchill, David (NBCC) ***@***.***>, Comment ***@***.***>
Subject: Re: [Softal-io/presentify-community] Ideas for Presentify App (Discussion #1)

This email was generated outside of NBCC, please use caution when replying.

You can erase selected drawings instead of everything. Please watch this video to see how - https://youtu.be/-fRLYxp7Ak4?si=JBsa5Ym1geSNVOiH

Regarding the opacity, from what I am reading, it seems you have enabled the Highlight mode unknowingly. Please, press H to disable it (when in annotation mode). This will give you 100% opaque drawings.

**Comment by dpac34-cmd** [comment]
Kindly add an icon on the toolbar clicking where and turn off the marker and then clicking the marker can turn the marker on without closing the toolbar . Like the feature in Screen Marker and recorder in windows.

**Reply by rampatra** [reply]
Good suggestion, would you mind adding this as a separate feature request?

**Comment by your-diary** [comment]
Layer functionality would help me a lot.
(Add layer, delete layer, hide layer, etc.)

**Comment by Arif-TEO** [comment]
Suggestion – Add Teleprompter Feature

You should consider adding a teleprompter feature. It would significantly enhance the overall presentation experience by helping presenters stay structured, confident, and on message without breaking flow.

**Reply by rampatra** [reply]
Tnx for the suggestion. Will build this as a separate app. I have some nice ideas for this app.

**Comment by atlasautomates** [comment]
Hey there Ram!  Great app; I use it every day now during presentations and for recording vids for clients. I really appreciate how seamless it is to turn on and off.

IDEA: Make Presentify usable even if not displayed in the menu bar. I tend to enjoy minimizing the clutter in the menu bar, but I can't seem to use Presentify if I remove it from the menu bar from the system settings. I'll attach a quick vid explaining what I mean.


https://github.com/user-attachments/assets/02cbae99-97a2-4f3d-82c9-d6b47004fb01

**Reply by rampatra** [reply]
Yes, currently Presentify is a menu bar only app. I can add an option to optionally hide the menu bar icon. This way the dock icon will pop up. Could you please kindly add this as a feature request?

**Reply by rampatra** [reply]
Just fyi, Presentify version 8.1.0+ has the option to disable the menu bar icon. You can use the Dock icon when you do so.

<img width="1731" height="1384" alt="CleanShot 2026-06-21 at 12 42 12@2x" src="https://github.com/user-attachments/assets/144a0a5b-e94d-471c-bba2-ecc786fdfee2" />

**Comment by DaveLee-fun** [comment]
**Feature request: live screen zoom**

I record coding tutorials and use Presentify for cursor highlight and drawing. The cursor ring is great and my students say it helps a lot.

The one thing I'm missing is live screen zoom. During a tutorial, I often need to zoom into a small section of code or a UI element so viewers can actually read it. Right now I use a separate app (ZoomShot) just for the zoom part, but it means I have two overlay tools running at the same time, which occasionally conflicts.

Would it be possible to add a keyboard-triggered screen zoom? Something like hold a hotkey and scroll to zoom in/out around the cursor area. Combined with Presentify's existing cursor highlight and drawing, that would make it a complete presentation toolkit and I could drop the second app.

Just putting it out there since I saw @WaffleJim and others also do live demos. Zoom would round out the feature set nicely.

**Reply by rampatra** [reply]
Presentify already has Zoom feature. Please update the app to the latest version.
<img width="2880" height="1800" alt="app-store-zoom" src="https://github.com/user-attachments/assets/1cc4df58-2f5c-4bd7-995f-ad67e7f98173" />

**Comment by MacchinaTech** [comment]
I'd love to see an option to reverse the "interact with underlying apps" feature, ie, I can use my apps as normal, then press and hold 'fn' and annotate until I release 'fn' and go back to the apps. 

Also, whenever I use the "interact with underlying apps" feature currently, I have to click on the underlying app to be able to interact with it using the keyboard - example would be a video player, I can't just press 'fn', and then hit the space bar to resume playing, I have to press the 'fn' key, click on the window, and then press space bar. I'd love to see some more fluid interaction between annotating and usage of apps.

Thanks, great app - Just these couple of improvements would make it perfect for me!

**Reply by rampatra** [reply]
Hey, both of these are addressed in the app. Please try out the latest version and comment here if you still have issues/questions.

**Reply by MacchinaTech** [reply]
Amazing, thank you! 

Both work well - the only issue is with interactive mode turned on, you can't undo or delete annotations (via 'cmd-z' or 'Del') without exiting interactive mode, because you're holding down 'fn' to interact, and that changes the modifier keys. Fixing this isn't critical for my specific use, but it's worth highlighting.

Love the update, thank you again!

**Comment by pnevesIPCB** [comment]
Hello!

Loving the apps so far. I bought the bundle with the 3 on App Store.

I'm using a Huion tablet to write on the screen. Is there any way I can map one of the pen keys to erase?

It seems to delete a single item I have to select it first with the command key. Am I missing something or there is not another way to directly delete a part of the drawing?

All the best,
Paulo Neves from Portugal

**Comment by DaveLee-fun** [comment]
Hi,
Thank you for your opinion.

First, if a specific pen action can be mapped to a particular shortcut key
or keyboard input, then it should already be possible to map it as a
shortcut combined with keys like Ctrl or Option.

Since there are so many types of pens, mapping specific pen actions
directly to specific keyboard keys is something you might want to review by
going through the pen documentation and giving it some thought.

Regarding memos,
the original intention was that when users write memos on the screen, they
may create multiple notes, and in many cases, they would want to delete
only some of them.

So, it seems more reasonable to allow users to delete individual memos by
clicking an “X” on each one.

If we were to introduce a shortcut for deleting memos, then when multiple
memos exist, it would likely delete all of them at once.
If we instead tried to support deleting a specific memo via a shortcut, the
user would first need to select that memo and then press the shortcut to
delete it, which could result in a rather awkward UX.

Adding a shortcut for deleting all memos could be provided as an option. If
needed, please let me know and we can consider including it in a future
version.

Best regards

Dave Lee

2026년 3월 20일 (금) AM 12:41, pnevesIPCB ***@***.***>님이 작성:

> Hello!
>
> Loving the apps so far. I bought the bundle with the 3 on App Store.
>
> I'm using a Huion tablet to write on the screen. Is there any way I can
> map one of the pen keys to erase?
>
> It seems to delete a single item I have to select it first with the
> command key. Am I missing something or there is not another way to directly
> delete a part of the drawing?
>
> All the best,
> Paulo Neves from Portugal
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Softal-io/presentify-community/discussions/1#discussioncomment-16210883>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOZL6QXCVQ64H2PCFEFNKVL4RQIJZAVCNFSM6AAAAACF2GN46KVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMMRRGA4DQMY>
> .
> You are receiving this because you commented.Message ID:
> <Softal-io/presentify-community/repo-discussions/1/comments/16210883@
> github.com>
>


-- 

잔재미코딩 Dave Lee 드림

잔재미코딩 유투브: ***@***.***

**Comment by pnevesIPCB** [comment]
Hello Dave,

Thanks for getting back to me so fast!

Is there any keyboard shortcut for erase function in annotations? My pen tablet can map any keyboard shortcut.

As for individual items deletion I was trying to refer to single pieces of annotations (like you can do on zoom whiteboard, just erase a single line) instead of erasing everything.

All the best,
Com os melhores cumprimentos,
[http://cdn.mcauto-images-production.sendgrid.net/2542a08d8df4cee6/2e3a1857-3785-4ac6-a9b2-d911f23055c2/639x659.png]
Paulo Alexadre Neves
GA.8, Professor Adjunto (MsC)
Campus da Talagueira
Avenida do Empresário
6000-767 Castelo Branco
+351 272 339 300
***@***.*** | www.ipcb.pt
[http://cdn.mcauto-images-production.sendgrid.net/2542a08d8df4cee6/5781b1b9-268f-4b7a-b599-abe1c1bb31b2/400x79.png]


On 20 Mar 2026, at 01:46, DaveLee ***@***.***> wrote:

﻿
AVISO DE SEGURANÇA: A origem desta mensagem é externa ao IPCB. Não clique em links nem abra anexos sem se certificar que o remetente e o conteúdo são seguros.


Hi,
Thank you for your opinion.

First, if a specific pen action can be mapped to a particular shortcut key
or keyboard input, then it should already be possible to map it as a
shortcut combined with keys like Ctrl or Option.

Since there are so many types of pens, mapping specific pen actions
directly to specific keyboard keys is something you might want to review by
going through the pen documentation and giving it some thought.

Regarding memos,
the original intention was that when users write memos on the screen, they
may create multiple notes, and in many cases, they would want to delete
only some of them.

So, it seems more reasonable to allow users to delete individual memos by
clicking an “X” on each one.

If we were to introduce a shortcut for deleting memos, then when multiple
memos exist, it would likely delete all of them at once.
If we instead tried to support deleting a specific memo via a shortcut, the
user would first need to select that memo and then press the shortcut to
delete it, which could result in a rather awkward UX.

Adding a shortcut for deleting all memos could be provided as an option. If
needed, please let me know and we can consider including it in a future
version.

Best regards

Dave Lee

2026년 3월 20일 (금) AM 12:41, pnevesIPCB ***@***.***>님이 작성:

> Hello!
>
> Loving the apps so far. I bought the bundle with the 3 on App Store.
>
> I'm using a Huion tablet to write on the screen. Is there any way I can
> map one of the pen keys to erase?
>
> It seems to delete a single item I have to select it first with the
> command key. Am I missing something or there is not another way to directly
> delete a part of the drawing?
>
> All the best,
> Paulo Neves from Portugal
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Softal-io/presentify-community/discussions/1#discussioncomment-16210883>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOZL6QXCVQ64H2PCFEFNKVL4RQIJZAVCNFSM6AAAAACF2GN46KVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMMRRGA4DQMY>
> .
> You are receiving this because you commented.Message ID:
> <Softal-io/presentify-community/repo-discussions/1/comments/16210883@
> github.com>
>


--

잔재미코딩 Dave Lee 드림

잔재미코딩 유투브: ***@***.***

**Comment by maxim333555777w-ship-it** [comment]
I have idea - do you want discussing?

**Reply by rampatra** [reply]
Sure thing, you can start by raising a new feature request.

**Comment by nienpoch** [comment]
Would you consider to add this feature:
Let highlighter mode (h key) has its own line width and color setting, so that freehand draw pen and highlighter can have separate line width and color. Currently, I have to press option-ctrl-5 to make it thick enough after hitting h to get in highlighter mode; press option-ctrl-1 to make the line  width back to default thin line after leaving highlighter mode.

**Reply by rampatra** [reply]
Hey, please kindly raise this as a new feature request and if it gets more upvotes, I can do it. Tnx!
